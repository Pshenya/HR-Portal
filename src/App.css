*{
    margin: 0;
    box-sizing: border-box;
    outline: 0;
    border: 0;
    vertical-align: baseline;
}

html{
    background-color: #f4f4f4;
}

ol,ul{
    list-style: none;
}

div{
    display: block;
}

.h1, .h2, .h3, .h4, .h5, .h6, h1, h2, h3, h4, h5, h6{
    margin: 0;
}

.app{
    width: 100%;
    color: #000 ;
    -webkit-font-smoothing: antialiased ;
    background-color: #f4f4f4;
}
