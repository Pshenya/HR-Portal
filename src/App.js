import React from 'react';
import 'bootstrap/dist/css/bootstrap.min.css';
import './App.css';

import Header from "./Components/Header/header";
import News from "./Components/Main/News/news";

function App() {
    return (
        <React.Fragment>
            <Header/>
            <News/>
        </React.Fragment>
    );
}

export default App;
