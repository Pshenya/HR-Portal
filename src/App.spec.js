import React from 'react';
import {App} from './App';
import {shallow} from "enzyme";


describe('App', () => {
    describe('App initial', () => {
        const app = shallow(<App/>);
        it('App is initialized and rendered', () => {
            expect(app).toMatchSnapshot()
        });
    });
    describe('App loading', () => {
        const app = shallow(<App />);
        it('renders properly', () => {
            expect(App).toMatchSnapshot()
        });
        it('renders Loading', () => {
            expect(app.find('Loading')).toMatchSnapshot();
        })
    });
    describe('App renders with error', () => {
        const app = shallow(<App />);
        it('renders properly', () => {
            expect(app).toMatchSnapshot()
        });
        it('renders error', () => {
            expect(app.find('error')).toMatchSnapshot()
        })
    })
});