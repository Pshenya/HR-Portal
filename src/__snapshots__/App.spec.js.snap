// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`App App initial App is initialized and rendered 1`] = `ShallowWrapper {}`;

exports[`App App loading renders Loading 1`] = `ShallowWrapper {}`;

exports[`App App loading renders properly 1`] = `[Function]`;

exports[`App App renders with error renders error 1`] = `ShallowWrapper {}`;

exports[`App App renders with error renders properly 1`] = `ShallowWrapper {}`;
